subprojects/robotpy-wpilib/wpilib/src/rpy/Filesystem.inc

```diff
@@ -10,11 +10,20 @@ static fs::path getMainPath() {
   py::gil_scoped_acquire gil;
   py::dict locals;
   py::exec(R"(
```

Comment:
Would it be worth defining this in pure Python instead? This is a fair bit of embedded Python code now.

Reply:
This was used by some vendor dep I think?